backend/models/Presentation.js

```diff
@@ -37,10 +37,6 @@ const Schema = mongoose.Schema;
         id: {
```

Comment:
slides: {
type: Object,
required: true
}
}, {timestamps: true});
lets change slides to this. we can add validations later as our presentation data is more fleshed out